metaflow/task.py

```diff
@@ -280,25 +283,42 @@ def clone_only(
                 "task.clone_only needs a valid clone_origin_task value."
             )
         origin_run_id, _, origin_task_id = clone_origin_task.split("/")
+        # Update system logger context
+        # We don't update the context for the system monitor as the cardinality of this context is high
```

Comment:
to confirm - your custom monitor implementation ignores whatever context is set? since other monitor implementations may need a context to be set here.